### PR TITLE
Move module to tf 0.12

### PR DESCRIPTION
**Pull request by venturel**
Update code to tf 0.12

### CODE REVIEW

File: app/codedeploy.tf [hunk 1]
```diff
@@ -54,8 +56,17 @@ resource "aws_iam_policy" "deployer_policy" {
...
+compact([var.s3_bucket_arn]),
+),
+0,
+)}
   ]
 }
 EOF
-}
+
+    }
```
Comment (duboisph) [inline app/codedeploy.tf]: I odn't think the indentation is right here (it's weird for the whole EOF block actually)

File: notify-slack/main.tf [hunk 2]
```diff
@@ -36,59 +37,64 @@ resource "aws_iam_role_policy" "cd_sns_lambda_policy" {
   }]
 }
 EOF
+
 }
 
 resource "aws_sns_topic" "cd-sns-lambda" {
-  name = "cd-sns-lambda"
+name = "cd-sns-lambda"
```
Comment (duboisph) [inline notify-slack/main.tf]: wrong indentation

File: notify-slack/main.tf [hunk 3]
```diff
@@ -36,59 +37,64 @@ resource "aws_iam_role_policy" "cd_sns_lambda_policy" {
...
 }
 
 resource "aws_sns_topic_subscription" "lambda-subscription" {
-  topic_arn = "${aws_sns_topic.cd-sns-lambda.arn}"
-  protocol  = "lambda"
-  endpoint  = "${aws_lambda_function.cd_sns_lambda.arn}"
+topic_arn = aws_sns_topic.cd-sns-lambda.arn
```
Comment (duboisph) [inline notify-slack/main.tf]: wrong indentation (the whole file actually)

**Review by duboisph — APPROVED** [review]
LGTM, after fixing indentation issues.

**Review by simonrondelez — APPROVED** [review]

**Pull request merged**